zerver/webhooks/honeybadger/view.py

```diff
@@ -0,0 +1,40 @@
+from datetime import datetime
+from typing import Any, Dict, Text
+
+from django.http import HttpRequest, HttpResponse
+
+from zerver.decorator import api_key_only_webhook_view
+from zerver.lib.actions import check_send_stream_message
+from zerver.lib.request import REQ, has_request_variables
+from zerver.lib.response import json_success
+from zerver.models import UserProfile
+
+@api_key_only_webhook_view('Honeybadger')
+@has_request_variables
+def api_honeybadger_webhook(request: HttpRequest, user_profile: UserProfile,
+                            payload: Dict[str, Any] = REQ(argument_type='body'),
+                            stream: Text = REQ(default='honeybadger')) -> HttpResponse:
+    body = '**{}**'.format(payload['message'])
+
+    if payload['event'] == 'check_in_missing':
+        reported_at = datetime.strptime(payload['check_in']['reported_at'], "%Y-%m-%dT%H:%M:%S.%fZ") \
```

Comment:
Also, could you please wrap this code at 80 characters per line? And I would suggest avoiding breaking up function calls over multiple lines with a `\`, like you do with `.strftime()`. :)